compiler/src/dmd/dsymbolsem.d

```diff
@@ -3387,11 +3416,11 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 )
                )
             {
-                funcdecl.flags |= (pors == SCOPE.printf) ? FUNCFLAG.printf : FUNCFLAG.scanf;
+                // no error
```

Comment:
Comment here is good but more detail required.

Reply:
Detail about what?

Reply:
You want me to explain how an if-else statement works?

Reply:
How about "the signature is valid for printf/scanf, no error", is that clearer?